TrafficCapture/replayerPlugins/jsonMessageTransformers/Es2OsPost23/src/test/java/org/opensearch/migrations/transform/Es2OsPost23Test.java

```diff
@@ -65,4 +65,4 @@ private static void transformAndVerifyResult(Map<String,Object> json, String exp
     static IJsonTransformer getJsonTransformer() {
         return new JsonTypeMappingTransformer();
```

Comment:
You should be testing the new transformer & you should be deleting this one, right?

Reply:
Added a new one that uses JMES transformers.